### DISCUSSION TITLE
[Bug]: Unable to Manually Activate Personal License on Company Jetbrains Account

### DISCUSSION THREAD

**Discussion by muya** [AtomMaterialUI/material-theme-issues#544] [Q&A] [answered]
### Describe the bug

I would like to use my personal license for Material Theme with IntelliJ which is logged in to my company account.

There is no option to manually enter the license details.

### To Reproduce

Following the instructions on license activation here: https://material-theme.com/docs/faq-troubleshooting/#license-activation

1. Open the Run Actions Modal
2. Search for "material license"
3. Choose "Activate License"

_Actual Behaviour_

- A prompt asking to restart the IDE to activate the license

https://github.com/user-attachments/assets/1bb81f83-8746-491d-bb32-b33d8bd4e123

### Expected behavior

Get a window where I can activate my license manually, as outlined in the guide here https://material-theme.com/docs/faq-troubleshooting/#license-activation

### Material Theme Version

v9.3.1

### IDE

IntelliJ IDEA

### System Info

```shell
IntelliJ IDEA 2024.1.4 (Ultimate Edition)
Build #IU-241.18034.62, built on June 20, 2024
Runtime version: 17.0.11+1-b1207.24 aarch64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
macOS 14.6
GC: G1 Young Generation, G1 Old Generation
Memory: 12288M
Cores: 12
Metal Rendering is ON
Registry:
  ide.balloon.shadow.size=0
  editor.focus.mode.color.light=2e5861
  editor.focus.mode.color.dark=2e5861
  debugger.new.tool.window.layout=true
  suggest.all.run.configurations.from.context=true
  ide.new.editor.tabs.vertical.borders=true
  ide.animate.toolwindows=true
  ide.experimental.ui=true
  ide.editor.tab.selection.animation=true
Non-Bundled Plugins:
  com.jetbrains.plugins.ini4idea (241.18034.82)
  org.intellij.plugins.hcl (241.17890.8)
  IdeaVIM (2.15.2)
  com.intellij.plugins.watcher (241.14494.150)
  org.jetbrains.plugins.go-template (241.14494.150)
  name.kropp.intellij.makefile (241.14494.150)
  com.github.zzehring.intellijjsonnet (0.2.1)
  org.jetbrains.plugins.go (241.17011.2)
  Pythonid (241.18034.62)
  com.intellij.datagen (241.14494.150)
  String Manipulation (9.14.1)
  org.jetbrains.plugins.hocon (2024.1.0)
  com.jetbrains.plugins.jade (241.15989.10)
  uk.co.ben-gibson.remote.repository.mapper (4.4.0)
  com.mallowigi (97.0.0)
  mobi.hsz.idea.gitignore (4.5.3)
  org.intellij.scala (2024.1.25)
  com.github.copilot (1.5.17.6356)
  com.chrisrm.idea.MaterialThemeUI (9.3.1)
  net.ashald.envfile (3.4.2)
Kotlin: 241.18034.62-IJ
```

**Comment by mallowigi** [comment]
It's not possible. The license is associated with one account, you can't use it on a different account.

This is how the marketplace works. I invite you to connect to JetBrains for future inquiries.

**Comment by muya** [comment]
> It's not possible. The license is associated with one account, you can't use it on a different account.
> 
> This is how the marketplace works. I invite you to connect to JetBrains for future inquiries.

Hello @mallowigi. Thank you for the prompt response. 

On my previous computer, I was using my personal license with my work computer, because I was previously able to manually activate my license.

Does this mean that I have to purchase a separate license that is associated to my work account?

**Comment by mallowigi** [comment]
I suggest you talk to JetBrains concerning these issues. Maybe they can figure out a plan for you. But normally yes, the marketplace has no way of knowing that the two accounts are related. I think there is a way to share licenses but I'm not sure.

**Comment by faelin** [comment]
> It's not possible. The license is associated with one account, you can't use it on a different account.
> 
> This is how the marketplace works. I invite you to connect to JetBrains for future inquiries.

@mallowigi that is simply not true. If you look at the screenshot below, you can see that several plugins allow a "Download code for offline activation" option, which allows me to activate my plugin in my Company-licensed IDE. However, you can clearly see that Material Theme does not provide that option.

![Screenshot 2024-10-09 at 11 41 47 AM](https://github.com/user-attachments/assets/d9c0201b-cbbb-44ef-839b-999d9b04e573)

I understand that Material Theme does not want users to simply share their codes, but this isn't about sharing. This is one user, who has purchased a single-use license subscription, using their own license. There is no sharing involved.

In fact, of the 8 plugins I pay for, Material Theme (and its add-ons) is the ONLY plugin that I cannot do this with.

**Comment by faelin** [comment]
I have the same issue, but I decided to comment here instead of opening a new issue. Please either re-open this issue, or resolve the problem by making a downloadable activation code available. This project is wonderful, and I would like to keep using it, but currently I simply CANNOT due to organizational limitations (I do not have access to subscribe to this plugin using my company account).

**Comment by mallowigi** [comment]
![CleanShot 2024-10-09 at 19 51 19](https://github.com/user-attachments/assets/e87f99a3-2e29-4695-a764-bbfc0a910c4d)

Looks fine on my end (also this is the first time I've heard about this feature).

As I said above, I suggest you contact JetBrains for such issues, as it is outside my control.

By the way, I've searched a bit and stumbled upon this: https://www.reddit.com/r/Jetbrains/comments/1d8syue/how_to_enter_my_own_material_theme_ui_license_on/

So it does work for other people, but not for you. I suppose there is a condition that is not met, but as I said, I can't help you more than that.

**Comment by faelin** [comment]
Knowing that the link appears for others, that definitely confirms that it is a Jetbrains issue. Thank you for showing that screenshot, that helps a lot!

**Comment by faelin** [comment, marked as answer]
@muya I found the answer we need! From JetBrains Support: https://sales.jetbrains.com/hc/en-gb/articles/7654171906706-Linking-multiple-email-addresses-to-your-JetBrains-Account

> This feature can help you, if you have several JetBrains Accounts (such as a personal account, a work account, or a school account) with different licenses associated with each of them.
> 
> Instead of constantly switching between accounts, you can link several email addresses to a single JetBrains Account (let’s call it the primary account) to collect all your licenses in one place.
